R/get_terms.R

```diff
@@ -1,26 +1,26 @@
-#' An example function as expected by the `get_smq_fun` parameter of
+#' An example function as expected by the `get_terms_fun` parameter of
 #' `admiral::create_query_data()`
 #'
-#' @param smq_select An smq_select object defining the terms
+#' @param basket_select A basket_select object defining the terms
 #'
 #' @param version MedDRA version
 #'
 #' @param keep_id Should `QUERY_ID` be included in the output?
 #'
 #' @param temp_env Temporary environment
-get_smq_terms <- function(smq_select,
+get_smq_terms <- function(basket_select,
```

Comment:
`get_smq_terms()` and `get_sdg_terms()` should be replaced by a single function (e.g., `get_terms()`). The `type` argument should be added to the `get_terms()` function. Depending on this argument the terms are read from `admiral_smq_db` or `admiral_sdg_dg`.

Reply:
@bundfussr thanks for the review, please see updates